ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -126,12 +123,27 @@ function pr-nnstreamer-ubuntu-apptest-run-queue() {
         echo -e "[DEBUG] USB Camera device is not enabled. It is required by {nnstreamer_example_filter|nnstreamer_example_cam}."
         echo -e "[DEBUG] Enabling virtual cam camera..." 
 
+        # Install virtual camera device kernel module, if it is not installed.
+        if [[ ! -d ${REFERENCE_REPOSITORY}/v4l2loopback ]]; then
+            echo -e "[DEBUG] Install virtual camera device..."
+            pushd ${REFERENCE_REPOSITORY}
+            git clone https://github.com/umlaeute/v4l2loopback.git
+            popd
+        fi        
+
         # Create virtual camera device and change authority for all.
-        sudo modprobe v4l2loopback 
+        pushd ${REFERENCE_REPOSITORY}/v4l2loopback
+        make clean && make
+        sudo insmod /lib/modules/`uname -r`/kernel/drivers/media/media.ko
+        sudo insmod /lib/modules/`uname -r`/kernel/drivers/media/v4l2-core/videodev.ko
+        sudo insmod ./v4l2loopback.ko
         pushd /dev
```

Comment:
As I commented previously, we do not need to run "sudo chmod 777 video0" command. 
Please check it again finally in this PR. 
`[FYI] # Dependency of kernel modules: media.ko --> videodev.ko --> v4l2loopback.ko `

Reply:
Without "sudo chmod 777 video0", element can not use '/dev/video0' cause permission denied

Reply:
> Without "sudo chmod 777 video0", element can not use '/dev/video0' cause permission denied

It's not reasonable. anyway, let's use workaroudn (e.g., chmod 777).